src/index.ts

```diff
@@ -0,0 +1,28 @@
+import RacingGame from "./controller/RacingGame.ts";
+
+import GarageStore from "./stores/GarageStore.ts";
+
+import Garage from "./domain/Garage.ts";
+
+// 📚 항상 예외를 체이닝하고 절대로 원래 예외를 무시하지 않는다.
+// 📚 예외를 잡은 즉시 새로운 예외를 던진다. (예외 체이닝)
+// 📚 원래의 문제를 새로운 문제로 감싸서 함께 상위로 던진다.
+// 📚 가장 최상위 수준에서 오직 한번만 복구하세요.
+
+// 📚 필요한 의존성을 객체가 직접 생성하는 대신 우리가 ctor을 통해 의존성을 주입합니다.
+
+async function main() {
+  try {
+    const game = new RacingGame(new GarageStore(new Garage()));
```

Comment:
```suggestion
    const store = new GarageStore(new Garage());
    const game = new RacingGame(store);  
```
요렇게 가독성을 올려봐도 좋을 것 같네요 무조건 해야된다는 아니고 just 가벼운 제안임!

Reply:
데코레이터를 조합할 때 수가 많아지면 가독성이 떨어지는 점이 고민되긴 하더라고요!
확실히 한 번씩 끊어가니까 조금 가독성이 올라가는 느낌이네요~

반영했어요~

Reply:
203cae2665c974f241dc515ce2274f48a59c1584